storage/container_storage_interface/osd-persistent-storage-aws-efs-csi.adoc

```diff
@@ -1,6 +1,6 @@
 :_content-type: ASSEMBLY
 [id="osd-persistent-storage-aws-efs-csi"]
-= Setting up AWS Elastic File Service CSI Driver Operator
+= Setting up Amazon Elastic File Service CSI Driver Operator
```

Comment:
Revoked.

Reply:
@smunje1 It doesn't seem like this title is linking out to OCP resources or code that reference "AWS EFS". "Amazon EFS CSI driver" is the appropriate name to be used for the driver. Is there a reason I'm missing why this update should be revoked?

https://docs.aws.amazon.com/eks/latest/userguide/efs-csi.html